### PR TITLE
tests: deflake _fs_read_test.ts

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/19632

### CODE REVIEW

File: cli/tests/unit_node/_fs/_fs_read_test.ts [hunk 1]
```diff
@@ -159,16 +159,20 @@ Deno.test({
...
   name: "[std/node/fs] Read fs.read(fd, cb) signature",
+  ignore: true,
   async fn() {
+    const promise = deferred();
     const file = Deno.makeTempFileSync();
     Deno.writeTextFileSync(file, "hi deno");
     const fd = openSync(file, "r+");
-    await read(fd, (err, bytesRead, data) => {
+    read(fd, (err, bytesRead, data) => {
       assertEquals(err, null);
```
Comment (dsherret) [inline cli/tests/unit_node/_fs/_fs_read_test.ts:169]: Will this test will hang if this throws?

File: cli/tests/unit_node/_fs/_fs_read_test.ts [hunk 2]
```diff
@@ -159,16 +159,20 @@ Deno.test({
 
 Deno.test({
   name: "[std/node/fs] Read fs.read(fd, cb) signature",
+  ignore: true,
```
Comment (dsherret) [inline cli/tests/unit_node/_fs/_fs_read_test.ts:162]: Why ignore?

Comment (bartlomieju) [reply cli/tests/unit_node/_fs/_fs_read_test.ts]: Too fast fingers :)

File: cli/tests/unit_node/_fs/_fs_read_test.ts (hunk 1 above)
Comment (bartlomieju) [reply cli/tests/unit_node/_fs/_fs_read_test.ts]: Reworked to reject on error.

**Review by dsherret — APPROVED** [review]
LGTM

**Pull request merged**